### ISSUE TITLE
Is it possible to change the height of an image box? 3.0

### ISSUE THREAD

**Issue by Goshimy** [garybentley/quollwriter#25]
In version 3.0.1 images themselves scale correctly, but not the image box. The vertical space they take up in a column remains whatever the original image size was. This makes it hard to add larger images as profile pictures for characters, I do not believe this behavior was in previous 2.1.6 version, but I am not able to verify. 

Attached examples image (which is a gif, but other images formats do the same thing) -

Image 1 - Original size of the image, no scale adjustments.
Image 2 - Column stretched horizontally expands the image box, but does not re scale image. or change image box height. 
Image 3 - Column squeezed horizontally image itself is scaled, but the original height of the image box remains unchanged. 
Image 4 - Same issue, squeezed horizontally the image scales with column width, but the image box remains same height as original image.

![1 - Original image size in column - Copy](https://user-images.githubusercontent.com/119022291/220967703-91a118b1-d66b-4724-9ef0-642d63ddae15.png)
![2 - Horizontal stretch in column - Copy](https://user-images.githubusercontent.com/119022291/220967717-dc4b47f7-3754-4016-9ae2-26a84843c383.png)
![3 - Horizontal squeez in column1 - Copy](https://user-images.githubusercontent.com/119022291/220967728-f14388ec-d4b6-4fb2-b43b-ea42f855db11.png)
![4 - Horizontal squeez in column2 - Copy](https://user-images.githubusercontent.com/119022291/220967749-99a33f81-ee7d-4fb6-b4cb-e8348e6f1c33.png)

**Comment by sixro** [comment]
Hi, how can you add an image?
I followed an old article on wordpress blog of Quoll Writer, but when I add a new asset, it doesn't allow me adding fields other than _Name_ and _Description_...

**Comment by garybentley** [comment]
@Goshimy yes I'm still working on trying to figure out how to scale properly in all circumstances.  It's on my list to fix.

**Comment by garybentley** [comment]
@sixro just right click on the asset and a menu should show up that allows you to add a new field.  Assets changed a lot from 2.6.14 to 3.  If you're still having trouble let me know.  The general rule for QW is right click on something and a menu of options should appear.
